doc/release_notes.rst

```diff
@@ -11,6 +11,10 @@ Release Notes
 Upcoming Release
 ================
 
+* Added option to use dynamic capacity for pit storage using the `e_max_pu` attribute of the store component, which is calculated in the new rule `build_tes_capacity_profiles` and added to the network in `prepare_sector_network`.
```

Comment:
```suggestion
* Added option to use dynamic capacity for pit storage using the ``e_max_pu`` attribute of the store component, which is calculated in the new rule :mod:`build_tes_capacity_profiles` and added to the network in :mod:`prepare_sector_network`.
```
RST syntax requires double-"`" for code. Rules you can refer to with `:mod:`<script_name>`.